integration/tests/epochs/suite.go

```diff
@@ -79,8 +82,8 @@ func (s *Suite) SetupTest() {
 	consensusConfigs := []func(config *testnet.NodeConfig){
 		testnet.WithAdditionalFlag("--hotstuff-timeout=12s"),
 		testnet.WithAdditionalFlag("--block-rate-delay=100ms"),
-		testnet.WithAdditionalFlag(fmt.Sprintf("--required-verification-seal-approvals=%d", 1)),
-		testnet.WithAdditionalFlag(fmt.Sprintf("--required-construction-seal-approvals=%d", 1)),
+		testnet.WithAdditionalFlag(fmt.Sprintf("--required-verification-seal-approvals=%d", 0)),
+		testnet.WithAdditionalFlag(fmt.Sprintf("--required-construction-seal-approvals=%d", 0)),
```

Comment:
Yes, Yurii noticed that with `require-approvals>0`, sealing temporarily stopped on some of the longer tests (and had a longer time-to-seal delay in general). The epoch tests in particular are pretty sensitive to this, because we have a pretty tight bound on the time to run the DKG, where we're waiting for blocks to be sealed at various points. 

We decided it was OK to turn it off for this test, since the MVP and verification tests do test the sealing flow requiring approvals. I'll add a comment explaining the decision.

Reply:
👍  thanks for the detailed explanation